src/main/java/io/lumigo/handlers/LumigoConfiguration.java

```diff
@@ -13,6 +13,7 @@
     private String token;
     private Boolean verbose;
     private Boolean killSwitch;
+    private Boolean lazyLoading;
```

Comment:
Why not initializing it with a default value of True and avoiding the null check at the top?

Reply:
Done